curator-recipes/src/main/java/org/apache/curator/framework/recipes/nodes/PersistentTtlNode.java

```diff
@@ -68,6 +69,7 @@ public class PersistentTtlNode implements Closeable {
     private final ScheduledExecutorService executorService;
     private final AtomicReference<Future<?>> futureRef = new AtomicReference<>();
     private final String childPath;
+    private volatile boolean skipTouch; // Just for testing
```

Comment:
I would prefer an overridable method(similar to the `deleteNode`) over unused member. Testing code could make it a nop to skip touch node creation.